app/controllers/admin/users_controller.rb

```diff
@@ -49,6 +49,14 @@ def user_status
       redirect_to "/admin/users/#{@user.id}/edit"
     end
 
+    def export_data
+      user = User.find(params[:id])
+      send_to_admin = JSON.parse(params[:send_to_admin])
+      ExportContentWorker.perform_async(user.id, send_to_admin: send_to_admin)
```

Comment:
BTW this is going to fail as Sidekiq doesn't support keyword arguments.

If you schedule the job you'll notice the following in Sidekiq:

![Screenshot_2020-09-11  DEVELOPMENT  Sidekiq](https://user-images.githubusercontent.com/146201/92885187-cd596d00-f412-11ea-97f7-6073c099ef13.png)